charts/policy-reporter/values.yaml

```diff
@@ -74,6 +74,9 @@ resources: {}
 # enable policy-report-ui
 ui:
   enabled: false
+  # Network Policy support, useful when running on a cluster configured with default deny policy
+  networkPolicy:
+    enabled: false
```

Comment:
You also need to define a third networkPolicy for the Kyverno Plugin. It needs to allow ingress traffic to 8080 and 2113 (Metrics)